app/move/controllers/assign/person-search-results.js

```diff
@@ -1,12 +1,13 @@
 const MoveCreatePersonSearchResultsController = require('../../../move/controllers/create/person-search-results')
 
-const PersonAssignBase = require('./base')
+const AssignToAllocationBaseController = require('./base')
 
-class PersonSearchResultsController extends PersonAssignBase {
+class PersonSearchResultsController extends AssignToAllocationBaseController {
   middlewareSetup() {
     super.middlewareSetup()
     this.use(this.setPeople)
     this.use(this.setPeopleItems)
+    this.use(this.setMoveSummary)
```

Comment:
Is this not already being called as part of `super.middlewareLocals`? I'd say that's probably the more logical place to actually make this call.

Reply:
Yup, good point.